src/components/GitTable.tsx

```diff
@@ -38,7 +42,7 @@ function GitTable() {
               <div className="align-center flex">
                 <img
                   src={commit.author?.avatar_url}
-                  alt="Avatar"
+                  alt={`${commit.author?.login} profile picture`}
```

Comment:
Maybe do this for safety if for some reason there is no author `alt={`${commit.author?.login ?? "User"} profile picture`}`